resources/views/backend/auth/user/includes/header-buttons.blade.php

```diff
@@ -1,3 +1,3 @@
-<div class="btn-toolbar float-right" role="toolbar" aria-label="@lang('labels.general.toolbar_btn_groups')">
-    <a href="{{ route('admin.auth.user.create') }}" class="btn btn-success ml-1" data-toggle="tooltip" title="@lang('labels.general.create_new')"><i class="fas fa-plus-circle"></i></a>
```

Comment:
User Management - Create User default functionality broken, as the link refers to admin.parking.create